py/nlr.h

```diff
@@ -64,8 +64,6 @@ struct _nlr_buf_t {
 #endif
 };
 
-extern nlr_buf_t *nlr_top;
```

Comment:
Removing this declaration makes the code below fail when MICROPY_NLR_SETJMP == 1. Should probably #include "py/mpstate.h" in that case?